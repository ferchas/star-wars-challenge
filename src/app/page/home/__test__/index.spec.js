import renderer from 'react-test-renderer';
import React from 'react';
import Home from '../index.js';

test('Home Page should render', () => {
  const component = renderer.create(
    <Home/>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
