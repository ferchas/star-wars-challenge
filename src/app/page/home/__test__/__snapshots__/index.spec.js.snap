// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render 1`] = `
<div
  className="layout_page"
>
  <header>
    <div
      className="header_title"
    >
      <span
        className="title"
      >
        TITLE
      </span>
      <span
        className="sub-title"
      >
        
      </span>
    </div>
    <div
      className="header_user"
    >
      <img
        className="icon-fa icon-user"
        src="./images/user.svg"
      />
      <span>
        Usuario
      </span>
    </div>
  </header>
  <div
    className="home-page"
  >
    <img
      className="menu"
      onClick={[Function]}
      src="./images/menu.png"
    />
    <nav
      className="close"
    />
    <article />
  </div>
</div>
`;
